client/signup/config/flows-pure.js

```diff
@@ -271,6 +271,17 @@ export function generateFlows( { getSiteDestination = noop, getPostsDestination
 		};
 	}
 
+	if ( config.isEnabled( 'signup/import-landing-handler' ) ) {
+		flows[ 'from-site' ] = {
+			steps: [ 'import-from-url', 'user', 'domains' ],
+			destination: () => '/checklist',
```

Comment:
Technically works, but since it's the importer flow, might be worth dropping users at the site importer:
```
destination: function( dependencies ) {
	return '/settings/import/' + dependencies.siteSlug;
},
```

Reply:
Good call!  Done in 65e4be9